example/lambdas/ftpPopulateTestLambda/index.js

```diff
@@ -85,24 +78,22 @@ const uploadFtpGranuleDataForDiscovery = async ({
   log.info(`Hostconfig is ${JSON.stringify(hostConfig)}`);
 
   const filePaths = await Promise.all(
-    ftpData.map(
-      async (file) => {
-        let objectKey;
-        try {
-          objectKey = await updateAndUploadTestFileToFtpHost({
-            file,
-            hostConfig,
-            prefix,
-            targetReplacementRegex: 'MOD09GQ.A2016358.h13v04.006.2016360104606',
-            targetReplacementString: newGranuleId,
-          });
-        } catch (error) {
-          log.info('Error!', error);
-          throw error;
-        }
-        return objectKey;
+    ftpData.forEach(async (file) => {
```

Comment:
Hm I don't believe this is functionally the same since `forEach` returns `undefined`, so we will be awaiting `undefined` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#return_value

On the other hand, `map` doesn't return `undefined` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#return_value 

Do you mind elaborating on this proposed change?

Reply:
Hello, this change was really a mistake. I have already reverted to the old code.
Thanks for the review.